src/renderers/client/render.tsx

```
@@ -0,0 +1,289 @@
+            }
+            // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+            this.saveAsIcon.current!.className = 'plotIcon hidden';
+            if (this.plotIcon.current) {
+                this.plotIcon.current.className = 'plotIcon hidden';
+            }
+            if (this.copyImageIcon.current) {
+                this.copyImageIcon.current.className = 'plotIcon hidden';
+            }
+        };
+        const contents = imgSrc ? (
+            <img src={imgSrc} alt="Cell output image" style={imgStyle} {...imgHeightWidth}></img>
```

Biome (2.1.2)

[error] Avoid the words "image", "picture", or "photo" in img element alt text.

Screen readers announce img elements as "images", so it is not necessary to redeclare this in alternative text.

(lint/a11y/noRedundantAlt)